packages/xchain-bitcoincash/src/utils.ts

```diff
@@ -76,3 +78,37 @@ export const encodeAddress = (address: string, network: string): string => {
 
   return getPrefix(network) + address
 }
+
+/**
+ * Parse transaction.
+ *
+ * @param {Transaction} tx
+ * @returns {Tx} Parsed transaction.
+ *
+ **/
+export const parseTransaction = (tx: Transaction): Tx => {
+  return {
+    asset: utils.AssetBCH,
```

Comment:
`AssetBCH` needs to be part of `xchain-utils` - see https://github.com/xchainjs/xchainjs-lib/issues/206